types/index.d.ts

```diff
@@ -187,3 +205,34 @@ type RuntimeOptions = {
   attributes: Record<string, string> | undefined;
 };
 type TODO = any;
+type CssModule = import("webpack").Module & {
+  content: Buffer;
```

Comment:
@alexander-akait can this `content` accept string or Buffer is the only possible value?

Reply:
We have `content: Buffer.from(content)` in our code, so I used Buffer, but webpack can accept strings too, need to tests, if it will work feel free to improve it